src/FINDOLOGIC/Api/Definitions/QueryParameter.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace FINDOLOGIC\Api\Definitions;
+
+class QueryParameter
```

Comment:
Would it be interesting to separate Smart Suggest parameter from others, or is having all in one place sufficiently understandable?

Reply:
Definitely